src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -114,13 +114,19 @@ export const getThorTradeQuote = async (
       slippageBps: thornodeQuote.slippage_bps,
       expectedAmountOutThorBaseUnit: thornodeQuote.expected_amount_out,
       isStreaming: false,
+      estimatedExecutionTimeMs:
+        1000 *
+        (thornodeQuote.inbound_confirmation_seconds ?? 0 + thornodeQuote.outbound_delay_seconds),
     },
     {
       // streaming swap
       source: `${SwapperName.Thorchain} • Streaming` as SwapSource,
       slippageBps: thornodeQuote.streaming_slippage_bps,
       expectedAmountOutThorBaseUnit: thornodeQuote.expected_amount_out_streaming,
       isStreaming: true,
```

Comment:
Not related to this PR, but in addition to @0xApotheosis comment https://github.com/shapeshift/web/pull/5118#issuecomment-1685797226

> We should probably hide the "percentage difference" when the quotes are identical.

Not related to this PR per se but wondering if we want to conditionally spread this guy depending on whether or not the non/streaming received amounts are different?